### PR TITLE
Added HighStoneSelector

### PR DESCRIPTION
**Pull request by Kjusok**
- Added screen with two buttons

### CODE REVIEW

File: Assets/Prefabs/UI/HighStoneChooseMenu/HighStoneChooseMenu.prefab
Comment (ega-ega) [inline Assets/Prefabs/UI/HighStoneChooseMenu/HighStoneChooseMenu.prefab:1]: Move this file to Prefabs/UI/Popups/ . And remove folder for this prefab

File: Assets/Prefabs/UI/StartSplashScreen/StartSplashScreen.prefab
Comment (ega-ega) [inline Assets/Prefabs/UI/StartSplashScreen/StartSplashScreen.prefab:1]: Move this file to Prefabs/UI/Popups/ . And remove folder for this prefab. Bulat already moved this file to Popup folder

**Review by ega-ega — CHANGES_REQUESTED** [review]

File: Assets/Prefabs/UI/StartSplashScreen/StartSplashScreen.prefab
Comment (Kjusok) [reply Assets/Prefabs/UI/StartSplashScreen/StartSplashScreen.prefab:1]: Done

File: Assets/Prefabs/UI/HighStoneChooseMenu/HighStoneChooseMenu.prefab
Comment (Kjusok) [reply Assets/Prefabs/UI/HighStoneChooseMenu/HighStoneChooseMenu.prefab:1]: Done

**Review by ega-ega — APPROVED** [review]

**Pull request merged**